### PR TITLE
New Theme + optimize images

### PR DESCRIPTION
**Pull request by nexogone**
### Warframe Hub Pull Request
---
---

**Summary (short):**

- New dark theme **(Eidolon)**
- Moving theme icons and backgrounds to **img/themes**
- Reducing images file sizes: **32MB -> 24,5MB**

---

(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

**Comment by TobiTenno** [comment]
eidolon theme doesn't work

**Comment by TobiTenno** [comment]
I'd add logic to force a grid update, since this forces everything down a bit and on the set of components i'm using, some things are floating down behind the footer (which doesn't happen if you force a repack)

**Comment by TobiTenno** [comment]
still need to force update grid after clicking the theme switcher for this

**Comment by TobiTenno** [comment]
@NeXoGone do you need help with it?

**Comment by nexogone** [comment]
@TobiTenno yes... I do not understand what I need to add ._.

**Comment by TobiTenno** [comment]
It's all good. you should just need to add a function call `updateGrid();` after [this if/else block](https://github.com/NeXoGone/warframe-hub/blob/cb05bfa1ca7c66d66477c3e1033f9e939e5923e9/views/partials/eidolon-theme.hbs#L17)

**Comment by TobiTenno** [comment]
looks good

**Review by TobiTenno — APPROVED** [review]

**Comment by TobiTenno** [comment]
@EricSihaoLin or @Tithen-Firion if either of you could approve as well, i just like more than one so we know it's been reviewed.

File: views/partials/eidolon-theme.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<script>
+	$(function() {
```
Comment (Tithen-Firion) [inline views/partials/eidolon-theme.hbs]: replace tabs with spaces

**Review by Tithen-Firion — CHANGES_REQUESTED** [review]

**Review by EricSihaoLin — APPROVED** [review]
I have briefly looked through the changes, I have no problems with it. 

Conditional approve: Tabs are standardized to 2 spaces to save space on screen.

File: views/partials/eidolon-theme.hbs [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-				theme.attr('href', 'css/bootstrap-default.min.css');
-				themecustom.attr('href', 'css/main.eidolon.css');
-				theme.removeClass();
-				theme.addClass('eidolon');
-				Cookies.set('mode', 'eidolon');
-			}
-			updateGrid();
-		});
+$(function() {
+	$('#eidolon').click(function() {
```
Comment (TobiTenno) [inline views/partials/eidolon-theme.hbs]: if it's spaces now, it needs to be 2 instead of.... 8

Comment (TobiTenno) [reply views/partials/eidolon-theme.hbs]: and it looks like it's still tabs

File: assets/css/main.eidolon.css [hunk 3]
```diff
@@ -405,10 +418,14 @@ img {
...
 }
 
+/* mobile width fix */
 @media only screen and (max-width: 768px) {
+	/* For mobile phones: */
   [class*="col-"] {
     width: 100%;
   }
+
+	/* For list items that go over labels */
```
Comment (Tithen-Firion) [inline assets/css/main.eidolon.css]: tab

File: views/partials/eidolon-theme.hbs (hunk 2 above)
Comment (Tithen-Firion) [inline views/partials/eidolon-theme.hbs]: more tabs in this file

**Review by Tithen-Firion — CHANGES_REQUESTED** [review]

**Review by Tithen-Firion — APPROVED** [review]

**Pull request merged**